subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaLibraryPlugin.java

```diff
@@ -19,7 +19,7 @@
 import org.gradle.api.Project;
 import org.gradle.api.plugins.internal.JavaPluginHelper;
 import org.gradle.api.plugins.jvm.JvmTestSuite;
-import org.gradle.jvm.component.internal.JvmSoftwareComponentInternal;
+import org.gradle.jvm.component.SingleTargetJvmFeature;
```

Comment:
💭 After seeing it everywhere in this Spike, I want to double down on my comments in the Spec about this name: Let's please not call it "SingleTarget".

Reply:
I agree most java libraries will not be multi-target. The default for java should be single target, though after a conversation with sterling, it seems making this API multi-target aware will be important. For this PR I think its fine to remove the prefix -- this is only a first iteration.